GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -747,7 +747,8 @@ private synchronized int writeInstanceData(
 
         Row firstRow = getRow(startRow, sheet);
 
-        writeMetadata(sheet, instanceData, startRow, (int) instanceData.maxIterationsCount, true);
+        //maxIterationsCount is actually the max iteration index; 0 for 1 iteration...
+        writeMetadata(sheet, instanceData, startRow, (int) instanceData.maxIterationsCount + 1, true);
```

Comment:
since we already pass in the `instanceData` parameter, do we need to pass in the `maxIterationsCount` as a parameter as well?

Reply:
Does this take into account the situations where there is a missing iteration index. e.g. we have iteration `0, 1, 3`?

Reply:
We never have that. instanceData will create empty filler iterations.

Reply:
And to answer the first question, for split sheets we call with another count.